src/dmd/complex.d

```diff
@@ -14,7 +14,7 @@ module dmd.complex;
 
 import dmd.root.ctfloat;
 
-struct complex_t
```

Comment:
You'll need to make most of these op\* methods `extern(D)`

Reply:
done

Reply:
Arguably its a DMD bug that that is rejected, but in the interest of moving things quickly...